src/resources/Microsoft.Authorization/policyDefinitions/Audit-AzureHybridBenefit-UnusedResourcesCostOptimization.json

```
@@ -0,0 +1,104 @@
+{
+    "name": "Audit-AzureHybridBenefit-UnusedResourcesCostOptimization",
```

```suggestion
    "name": "Audit-AzureHybridBenefit-UnusedResourcesCostOptimization",
```
I suspect this policy name may be too long. The policy name would be fine without the -UnusedResourcesCostOptimization bit. Have you successfully tested the deployment through ALZ?

---

Updated. The code has been tested elsewhere.